src/wazuh_db/wdb_parser.c

```diff
@@ -6020,6 +6027,19 @@ int wdb_parse_get_groups_integrity(wdb_t* wdb, char* input, char* output) {
     return OS_SUCCESS;
 }
 
+int wdb_parse_global_recalculate_agent_group_hashes(wdb_t* wdb, char* output) {
+
+    if (OS_SUCCESS != wdb_global_recalculate_all_agent_groups_hash(wdb)) {
+        mdebug1("Error recalculating group hash of agents in global.db.");
```

Comment:
I think we can change this log to `warning`.

Reply:
Done!